recipe/meta.yaml

```diff
@@ -35,17 +35,28 @@ test:
   imports:
     - xonsh
   commands:
-    - xonsh -h
+    - USER=test xonsh -h  # [unix]
+    - xonsh -h  # [win]
 
 app:
   entry: xonsh
   icon: xonsh.png
+  summary: xonsh is a Python-ish, BASHwards-facing shell.
+  type: desk
 
 about:
   home: http://xon.sh
   license: BSD
```

Comment:
That's what I found [here]( https://github.com/scopatz/xonsh/blob/master/license ).